utilities/decrypt_data/export_postgres_to_excel_chunks.py

```
@@ -0,0 +1,585 @@
+            CONFIG['export']['use_chunking'], 
+            CONFIG['export']['chunk_size']
+        )
+        
+        # Print detailed summary report
+        exporter.print_export_summary(export_summary)
+        
+        if export_summary['total_tables_exported'] > 0:
+            if export_summary['data_integrity_check']:
+                print(f"\nSUCCESS: Export completed successfully. All {export_summary['total_db_rows']:,} records exported to '{OUTPUT_DIR}' directory")
+            else:
+                print(f"\nWARNING: Export completed with data integrity issues. Please check the summary above.")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)